OrbitVulkanLayerClientGgp/layer_config.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+package orbit_vulkan_capture_protos;
+
+message LayerConfig {
+  CaptureServiceArguments capture_service_arguments = 1;
+
+  LayerOptions layer_options = 2;
+}
+
+message CaptureServiceArguments {
+  repeated string functions = 1;  // Functions to hook to the capture
+
+  string file_directory =
```

Comment:
Formatting looks really weird here. I would put the comment above the line to avoid the line break. 

```
// Path to locate .orbit file...
string file_directory = 2;
```

Reply:
Fixed